packages/ffmpeg.rb

```diff
@@ -1,121 +1,217 @@
+             {"enable_transform",  "Enables transform for computing vmaf.",                      OFFSET(enable_transform), AV_OPT_TYPE_BOOL, {.i64=0}, 0, 1, FLAGS},
+      VMAF_PATCH_EOF
+      IO.write('vmaf.patch', @vmaf_patch)
+      system 'patch -p1 -i vmaf.patch'
+      # ChromeOS awk employs sandbox redirection protections which screw
+      # up configure script generation, so use mawk.
+      system "sed -i 's/awk/mawk/g' configure"
+      system "env CFLAGS='-pipe -fno-stack-protector -U_FORTIFY_SOURCE #{@lto} -fuse-ld=gold' \
+        CXXFLAGS='-pipe -fno-stack-protector -U_FORTIFY_SOURCE #{@lto} -fuse-ld=gold' \
+        LDFLAGS='-fno-stack-protector -U_FORTIFY_SOURCE #{@lto}' \
+        CC=clang CXX=clang++ \
+        ./configure \
+        --arch=#{ARCH} \
+        #{CREW_OPTIONS.sub(/--build=.*/, '')} \
+        --disable-debug \
+        --enable-avisynth \
+        --enable-ffplay \
+        --enable-fontconfig \
+        --enable-frei0r \
+        --enable-gmp \
+        --enable-gnutls \
+        --enable-gpl \
+        --enable-ladspa \
+        --enable-libaom \
+        --enable-libass \
+        --enable-libbluray \
+        --enable-libdav1d \
+        --enable-libdrm \
+        --enable-libfontconfig \
+        --enable-libfreetype \
+        --enable-libfribidi \
+        --enable-libgsm \
+        --enable-libiec61883 \
+        --enable-libjack \
+        #{@mfx}  \
+        --enable-libmodplug \
+        --enable-libmp3lame \
+        --enable-libopencore_amrnb \
+        --enable-libopencore_amrwb \
+        --enable-libopenjpeg \
+        --enable-libopus \
+        --enable-libpulse \
+        --enable-librav1e \
+        --enable-librtmp \
+        --enable-librubberband \
+        --enable-libsnappy \
+        --enable-libsoxr \
+        --enable-libspeex \
+        --enable-libsrt \
+        --enable-libssh \
+        --enable-libtesseract \
+        --enable-libtheora \
+        --enable-libv4l2 \
+        --enable-libvidstab \
+        --enable-libvmaf \
+        --enable-libvorbis \
+        --enable-libvpx \
+        --enable-libwebp \
+        --enable-libx264 \
+        --enable-libx265 \
+        --enable-libxcb \
+        --enable-libxml2 \
+        --enable-libxvid \
+        --enable-libzmq \
+        --enable-libzimg \
+        #{@enablelto} \
+        --enable-lzma \
+        --enable-nonfree \
+        --enable-pthreads \
+        --enable-shared \
+        --enable-version3 \
+        --host-cflags='-pipe -fno-stack-protector -U_FORTIFY_SOURCE #{@lto} -fuse-ld=gold' \
+        --host-ldflags='-fno-stack-protector -U_FORTIFY_SOURCE #{@lto}'"
+
+      system "env PATH=#{CREW_LIB_PREFIX}/ccache/bin:#{CREW_PREFIX}/bin:/usr/bin:/bin \
+        make"
+      system 'make tools/qt-faststart'
+      system 'make doc/ffmpeg.1'
+      system 'make doc/ffplay.1'
```

Comment:
We need `make` for manpages?

Reply:
See https://github.com/archlinux/svntogit-packages/blob/8ab4ef09f42f02fce532722c6581508d3baecaf3/trunk/PKGBUILD#L176

Reply:
What do we need to make?  Doesn't make call "functions" in the Makefile?

Reply:
Looking at https://github.com/FFmpeg/FFmpeg/blob/master/doc/Makefile the rationale appears to be to only build those two man pages instead of all possible documentation.